packages/relay/src/lib/eth.ts

```diff
@@ -1605,7 +1636,6 @@ export class EthImpl implements Eth {
    * @param returnLatest
    */
   private async getHistoricalBlockResponse(blockNumberOrTag?: string | null, returnLatest?: boolean, requestId?: string | undefined): Promise<any | null> {
```

Comment:
`blockNumberOrTag` is possibly the same type as `blockParam` i.e.:
```
type BlockParam = {
  blockHash: string
} | string | null;
```

This issue likely affects all of the methods listed in [EIP-1898](https://eips.ethereum.org/EIPS/eip-1898).

Reply:
Correct. Captured in https://github.com/hashgraph/hedera-json-rpc-relay/issues/822